### PR TITLE
feat: added `autoFocus` prop to Text, Paragraph and Title in Typography on editable input

### PR DESCRIPTION
**Pull request by OmriGM**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/25811
### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/typography/index.en-US.md](https://github.com/OmriGM/ant-design/blob/OmriGM-typography/components/typography/index.en-US.md)
[View rendered components/typography/index.zh-CN.md](https://github.com/OmriGM/ant-design/blob/OmriGM-typography/components/typography/index.zh-CN.md)

### CODE REVIEW

File: components/typography/index.zh-CN.md [hunk 1]
```diff
@@ -47,6 +48,7 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | underline | 添加下划线样式 | boolean | false |  |
 | onChange | 当用户提交编辑内容时触发 | function(string) | - |  |
 | type | 文本类型 | `secondary` \| `warning` \| `danger` | - |  |
+| autoFocus | 自动聚焦文本区域 (仅在 “可编辑” 为 true 时有效) | boolean | false |  |
```
Comment (07akioni) [inline components/typography/index.zh-CN.md:51]: `editable` 为 true

add 4.6.0 to version column

File: components/typography/index.zh-CN.md [hunk 2]
```diff
@@ -63,6 +65,7 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | onChange | 当用户提交编辑内容时触发 | function(string) | - |  |
 | strong | 是否加粗 | boolean | false |  |
 | type | 文本类型 | `secondary` \| `warning` \| `danger` | - |  |
+| autoFocus | 自动聚焦文本区域 (仅在 “可编辑” 为 true 时有效) | boolean | false |  |
```
Comment (07akioni) [inline components/typography/index.zh-CN.md:68]: 同上

**Review by 07akioni — COMMENTED** [review]
submit to feature branch

**Comment by zombieJ** [comment]
https://github.com/ant-design/ant-design/issues/25811#issuecomment-667790926

**Comment by OmriGM** [comment]
> [#25811 (comment)](https://github.com/ant-design/ant-design/issues/25811#issuecomment-667790926)

So this pull request isn't relevant anymore?

**Comment by zombieJ** [comment]
@OmriGM, you can create another PR or update this PR also : )

**Comment by OmriGM** [comment]
@zombieJ What I meant is that if we can achieve the auto focus capability with: 

``` <Typrograpy.Text editable={ editing: true }> ``` 

Is it really necessary to add an `autoFocus` prop? :)

**Comment by zombieJ** [comment]
No need to add `autoFocus`. Auto trigger focus with `editing` is good enough : )

**Comment by OmriGM** [comment]
@zombieJ Yes that's exactly what I meant :), so the prop addition I made is not relevant and there is no such need for any change to the code. So I'll close this PR.

**Comment by zombieJ** [comment]
Got it. Pls feel free if you willing to create another PR : )

**Pull request closed without merging**